client/src/components/kanbancomponents/KanbanForm.js

```diff
@@ -0,0 +1,103 @@
+import Tag from '../kanbancomponents/KanbanTask';
+import { useState } from 'react';
+
+
+export default function KanbanForm({ submitFunction }) {
+    const initialColumn = {
+        column_name: '',
+        issue_name: '',
+        task: [],
+        urgency: ''
+    }
+
+    const [column, setColumn] = useState(initialColumn);
+
+
+    //console.log(column)
+    //console.log(columns)
+
+    const handleChange = (event) => {
+        const field = event.target; //wo ich es eingebe
+        const value = field.value; //was ich eingebe, in value soeichere ich das rein was ich eingegeben habe
+        setColumn({
+            ...column,
+            [field.name]: value //um das in den namen einzuspeichern
+        })
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault();
+        submitFunction(column)
+        setColumn(initialColumn)
+    }
+
+    const addProductTag = (tag) => {
```

Comment:
Tag/Tasks? What is it :) Also check the function name again. What will be added here? A new task?